<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela_Principal</class>
 <widget class="QMainWindow" name="Janela_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>DiagBot</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="resposta_robo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(12, 158, 255);
border-radius : 15px;
font: 16pt &quot;Lucida Console&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pergunta_usuario">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(150, 158, 255);
border-radius : 15px;
font: 16pt &quot;Lucida Console&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botton_info">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(181, 181, 181);
border-radius : 20px; 
border : 2px solid grey;
font: 20pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>i</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>320</width>
      <height>260</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Background/Logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada_usuario">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>415</y>
      <width>451</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(181, 181, 181);
border-radius : 15px;
font: 16pt &quot;Lucida Console&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Logos/logo_mensagem.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="botton_info_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(181, 181, 181);
border-radius : 20px;
border : 2px solid grey;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>logo_mensagem.png</normaloff>logo_mensagem.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>resposta_robo</zorder>
   <zorder>pergunta_usuario</zorder>
   <zorder>botton_info</zorder>
   <zorder>entrada_usuario</zorder>
   <zorder>label</zorder>
   <zorder>botton_info_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../Desktop/Daniel/Fundo.qrc"/>
  <include location="icones.qrc"/>
 </resources>
 <connections/>
</ui>
